lib/device.js

```diff
@@ -45,13 +49,20 @@ class Device {
     }
 
     if (options.scrambleKey != null) {
-      assert(typeof options.scrambleKey === 'string');
+      if (typeof options.scrambleKey === 'string')
+        options.scrambleKey = Buffer.from(options.scrambleKey, 'ascii');
+
+      assert(Buffer.isBuffer(options.scrambleKey),
+        'scramble key must be buffer');
+
       this.scrambleKey = options.scrambleKey;
```

Comment:
What is a scrambleKey for?

Reply:
It's used for u2f communication

Reply:
Do you have a link that explains how this is used? Is it like an API access token type of thing?